### PR TITLE
change namespace value to Release.Namespace

### PR DESCRIPTION
**Pull request by my-git9**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
The value of namespace should be written as `Release.namespace` as elsewhere, not a fixed value

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by wackxu** [comment]
lgtm

File: manifests/charts/cloudcore/templates/deployment_controllermanager.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
   annotations: {{- toYaml . | nindent 4 }}
   {{- end }}
   name: kubeedge-controller-manager
-  namespace: kubeedge
+  namespace: {{ .Release.Namespace }}
```
Comment (gy95) [inline manifests/charts/cloudcore/templates/deployment_controllermanager.yaml:12]: i can not find where Release.Namespace come from?

Comment (my-git9) [reply manifests/charts/cloudcore/templates/deployment_controllermanager.yaml:12]: It comes from the namespace where the chart was released to.

Comment (gy95) [reply manifests/charts/cloudcore/templates/deployment_controllermanager.yaml:12]: can you point it out in source codes? thanks

Comment (my-git9) [reply manifests/charts/cloudcore/templates/deployment_controllermanager.yaml:12]: Is the document ok? https://helm.sh/docs/topics/charts/#predefined-values
image

Comment (gy95) [reply manifests/charts/cloudcore/templates/deployment_controllermanager.yaml:12]: lgtm
/cc @zhu733756 @zc2638

**Comment by zc2638** [comment]
I have a question. Other resources are missing the specified namespace. 
need to add them?
@my-git9

**Comment by my-git9** [comment]
> I have a question. Other resources are missing the specified namespace. need to add them? @my-git9

Not specifying under `meta.namespace` has no effect, but the namespace of serviceaccout specified in rbac's rolebinding and clusterrolebinding must be specified. In other places about rbac, it is already `{{ .Release.Namespace }}`

**Pull request merged**